### ISSUE TITLE
Removal of GeoAPI version Prevents interop with NetTopologySuite

### ISSUE THREAD

**Issue by SamuelEnglard** [NetTopologySuite/ProjNet4GeoAPI#52]
After #49, both GeoAPI and this project have their own `IProjection`. Since NTS uses GeoAPI's I cannot use Projections from here with NTS.

**Comment by airbreather** [comment]
What specific packages are you using, including versions?

NTS 2.x will be compatible with ProjNet 2.x, since neither will use GeoAPI.

I think the 1.x version of either package should be compatible with the 2.x version of the other, but I can imagine that this is an area where we can improve.

**Comment by airbreather** [comment]
> What specific packages are you using, including versions?

A code sample and error message would also help a lot.

**Comment by SamuelEnglard** [comment]
> NTS 2.x will be compatible with ProjNet 2.x, since neither will use GeoAPI.

Currently using 1.15.0 but if NTS 2.x will also not use GeoAPI then problem solved.

No sample code yet, was investigating an idea and noticed this. It would be a type mismatch though.

**Comment by airbreather** [comment]
2.0.0 packages of everything are on NuGet now (for ProjNet, the package ID has [changed](https://www.nuget.org/packages/ProjNet/2.0.0)).

If you run into problems using those packages together, please open a new issue with the details.  Closing.
